src/config.py

```diff
@@ -241,9 +241,12 @@ def load_data():
             "atk_len_diablo_infector": float(Config._select_val("char", "atk_len_diablo_infector")),
             "atk_len_diablo": float(Config._select_val("char", "atk_len_diablo")),
             "atk_len_cs_trashmobs": float(Config._select_val("char", "atk_len_cs_trashmobs")),
-            "kill_cs_trash": float(Config._select_val("char", "kill_cs_trash")),
+            "kill_cs_trash": bool(Config._select_val("char", "kill_cs_trash")),
```

Comment:
kill_cs_trash boolean was being read as True, even when set to 0, making it impossible to run without killing trash. 

Observing that the other booleans in this dict have an int(), I added an int() to the entry.  
kill_cs_trash was interpreted as False and the script was skipping the trash as expected from kill_cs_trash=0

Suggested change:
kill_cs_trash": bool(int(Config._select_val("char", "kill_cs_trash"))),

My first discovery and microcontribution :) - Mischa

Reply:
Fixed! thanks!